### PR TITLE
[flink] Introduce RepairTable Action for flink

### PR DESCRIPTION
**Pull request by xuzifu666**
### Purpose
Currently Flink Action not support RepairTable，so introduce RepairTable Action for flink 

Linked issue: close [#xxx](https://github.com/apache/paimon/issues/3660)

### CODE REVIEW

**Comment by zhuangchong** [comment]
The action function input parameters should ideally be consistent with the `Procedure`.

```
-- repair all databases and tables in catalog
CALL sys.repair()

-- repair all tables in a specific database
CALL sys.repair('databaseName')

-- repair a table
CALL sys.repair('databaseName.tableName')
```

**Comment by xuzifu666** [comment]
> The action function input parameters should ideally be consistent with the `Procedure`.
> 
> ```
> -- repair all databases and tables in catalog
> CALL sys.repair()
> 
> -- repair all tables in a specific database
> CALL sys.repair('databaseName')
> 
> -- repair a table
> CALL sys.repair('databaseName.tableName')
> ```

Done @zhuangchong

File: paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/RepairAction.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            String tableName,
+            Map<String, String> catalogConfig) {
+        super(warehouse, catalogConfig);
+        this.databaseName = databaseName;
+        this.tableName = tableName;
+    }
+
+    @Override
+    public void run() throws Exception {
+        Preconditions.checkArgument(!tableName.contains("\\."), "tableName can't contain ','");
```
Comment (zhuangchong) [inline paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/RepairAction.java:47]: Should this verification be removed? 

There can be only one parameter `identifier`.

Comment (xuzifu666) [reply paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/RepairAction.java]: Yes，had removed it. @zhuangchong

**Review by zhuangchong — APPROVED** [review]
+1

**Comment by JingsongLi** [comment]
+1 thanks @xuzifu666 @zhuangchong

**Pull request merged**